db/patches/V1_6_61_12__add_user_ranking.sql

```diff
@@ -0,0 +1,2 @@
+-- Add a new user rank
+INSERT INTO user_rankings (rank_name) VALUE ('Grandmaster');
```

Comment:
Can you specify the id - I'm not sure if the auto increment will necessarily be at the right value

Reply:
I certainly can if it will cause a problem if I don't, but it is slightly inconvenient for me because it invalidates all my dev databases. Normally it's not a huge deal, but I made a lot of map/account changes after my pre-migration backup that would be tedious to redo.

When I applied the migration locally it worked fine:
![image](https://user-images.githubusercontent.com/846186/34454464-f83f27b8-ed20-11e7-9bf5-08819473879f.png)

Reply:
I don't know if it will or not, I just don't necessarily trust the state of the live db is all, so it might need some intervention when it gets deployed (or it might not)

Reply:
This is the table from the live server, so we should be good!
![image](https://user-images.githubusercontent.com/846186/34457649-e49d3812-ed6b-11e7-819a-ea752cc39d89.png)